libs/dal/src/repositories/topic/topic.entity.ts

```diff
@@ -9,4 +9,5 @@ export class TopicEntity {
   name: string;
   // For users to have related anything they want with the topic.
   customData?: Record<string, unknown>;
+  subscribers?: SubscriberId[];
```

Comment:
This approach could be our solution to the wrong mismatched between `subscriberId` and `_subscriberId` want to think about it on the cooldown?

Reply:
Not following you here. What problem are we facing right now?

Reply:
That today we make mistakes between subscriberId and _subscriberId, having a dedicated type for each instead of just a string can help solving that :)